packs/fixtures/sensors/test_passive_sensor.py

```diff
@@ -0,0 +1,63 @@
+# Requirements:
+# See ../requirements.txt
+# import datetime
+
+import json
+
+from flask import Flask
+
+from st2reactor.sensor.base import Sensor
+
+SAMPLE_PAYLOAD = {
+    'str': 'String',
+    'int': 1,
+    'boo': True,
+    'obj': {
+        'foo': 'bar',
+        'baz': 1
+    },
+    'lst': [1, 5, 7]
+}
+
+
+class TestPassiveSensor(Sensor):
+    def __init__(self, sensor_service, config=None):
+        super(TestPassiveSensor, self).__init__(sensor_service=sensor_service,
+                                                config=config)
+        self._trigger_pack = 'fixtures'
+        self._trigger_ref = '.'.join([self._trigger_pack, 'test_passive_trigger.dummy'])
+        self.host = self._config['host']
+        self.port = self._config['port']
+        self.app = Flask(__name__)
+
+    def setup(self):
+        @self.app.route('/webhooks/<path:endpoint>', methods=['POST', 'GET'])
```

Comment:
I know why you have done this and for testing that's fine as long as we don't reference this example in the documentation since it would be an anti-pattern.

I know there are some exceptions (e.g. new relic sensor), but in general, you should use webhook when possible if you are sending data to StackStorm over HTTP and you control the sending process.